pipelines/utilities/install_backupstores.sh

```diff
@@ -1,6 +1,35 @@
 install_backupstores(){
   MINIO_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/minio-backupstore.yaml"
   NFS_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/nfs-backupstore.yaml"
+  CIFS_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/cifs-backupstore.yaml"
+  AZURTIZE_BACKUPSTORE_URL="https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/backupstores/azurite-backupstore.yaml"
   kubectl create -f ${MINIO_BACKUPSTORE_URL} \
-                 -f ${NFS_BACKUPSTORE_URL}
+                 -f ${NFS_BACKUPSTORE_URL} \
+                 -f ${CIFS_BACKUPSTORE_URL} \
+                 -f ${AZURTIZE_BACKUPSTORE_URL}
+  setup_azuitize_backup_store
+}
+
+setup_azuitize_backup_store(){
+  RETRY=0
+  MAX_RETRY=60
+  until (kubectl get pods | grep 'longhorn-test-azblob' | grep 'Running'); do
+    echo 'Waiting azurtize pod running'
+    sleep 5
+    if [ $RETRY -eq $MAX_RETRY ]; then
+      break
+    fi
+    RETRY=$((RETRY+1))
+  done
+
+  AZBLOB_ENDPOINT=$(echo -n "http://$(kubectl get svc azblob-service -o jsonpath='{.spec.clusterIP}'):10000/" | base64)
+  kubectl -n longhorn-system patch secret azblob-secret \
+    --type=json \
+    -p="[{'op': 'replace', 'path': '/data/AZBLOB_ENDPOINT', 'value': \"${AZBLOB_ENDPOINT}\"}]"
+
+  CONTROL_PLANE_PUBLIC_IP=$(cat /tmp/controlplane_public_ip)
+  CONTROL_PLANE_PRIVATE_IP=$(kubectl get nodes -o wide | awk '/control-plane/ {print $6}')
+  # port forward and az container create need to be run on control node
+  ssh ec2-user@${CONTROL_PLANE_PUBLIC_IP} "nohup kubectl port-forward --address 0.0.0.0 service/azblob-service 20001:10000 > /dev/null 2>&1 &"
```

Comment:
Do you know why we need to ssh into the control plane to run the `az` command? Can it be directly run on the Jenkins agent like other `kubectl` commands to skip the `ssh` step?

Reply:
It should can perform remote create, but since we need to ssh into control plane node to perform port forward(the [workaround](https://confluence.suse.com/display/LON/Setup+Azurite+Backupstore+For+Testing) to setup azurite backupstore), using one ssh command to create az storage on control plane node directly is more efficient. I'd love to hear your thoughts on this, thank you.

Reply:
Isn't the port-forward workaround using a `local port`? Not sure but I assume it means your local machine rather than the control plane node? So do you mean if we port-forward on the local machine, the workaround will not work?

```
3. Forward azblob-service
Forward local port 20001 traffic to port 10000 on the service
```

Additionally, the username `ec2-user` in `ec2-user@${CONTROL_PLANE_PUBLIC_IP}` limits the script to `EC2 sles` or `sle-micro` instances. But it won't be work on other distros like rocky, rhel, etc., and other environments like local VM or harvester HAL cluster.

Reply:
Yes I did a try, if the port forward running on jenkins agent, the backup store can not be accessed. For the local port description, atually means the kubernetes node, I have updated the confluence content.